src/components/Auction/style.css

```diff
@@ -0,0 +1,67 @@
+.auction__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 1rem;
+}
+.auction__creator-info {
+  display: flex;
+  justify-content: space-between;
+}
+.auction__amount-bid {
```

Comment:
Toto v tom mobilním view taky začne přetýkat, buď se to dá vyřešit pomocí `flex-wrap: wrap`, aby se to hodilo na novej řádek. Ale možná je tohle dobrej usecase na ten grid + @media query